### PR TITLE
Fix unit test instructions

### PR DESCRIPTION
**Pull request by tsuz**
Fixes https://github.com/mapbox/mapbox-gl-native-android/issues/661

### CODE REVIEW

File: TESTS.md [hunk 1]
```diff
@@ -86,9 +86,12 @@ You can also have a run configuration:
 * Click on "Junit Tests"
 * Give a name to the configuration, e.g. `JUnit tests`
 * As "Test Kind", choose "All in directory"
-* As folder, choose the following folder: `mapbox-gl-native/android/java/MapboxGLAndroidSDKTestApp/src/test/java`
+* As folder, choose the following folder: `${PROJECT_DIR}/mapbox-gl-native/android/java/MapboxGLAndroidSDKTestApp/src/androidTest/java`
```
Comment (Chaoba) [inline TESTS.md:89]: The folder should be `mapbox-gl-native-android/MapboxGLAndroidSDKTestApp/src/androidTest/java`

**Review by Chaoba — COMMENTED** [review]
Thanks for your contribution.

**Comment by tsuz** [comment]
@Chaoba 

Hmm, I cannot run this... I get the below error.
```
Class not found: "com.mapbox.mapboxsdk.maps.MapboxTest"
```

Although, if I specify the specific class or method, then it runs with the below config:


Screen Shot 2021-04-13 at 23 08 02

**Comment by Chaoba** [comment]
@tsuz We don't have unit test in testApp module, so the path should be `mapbox-gl-native-android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk`

**Pull request closed without merging**